packages/grpc-js/src/server-interceptors.ts

```diff
@@ -39,6 +39,11 @@ function trace(text: string) {
   logging.trace(LogVerbosity.DEBUG, TRACER_NAME, text);
 }
 
+export interface GrpcFrame {
```

Comment:
This shouldn't be exported. It's only used in internal APIs in this file.

Reply:
Fair